impyute/dataset/base.py

```diff
@@ -1,6 +1,11 @@
 """ Shared functions to load/generate data """
 import numpy as np
+import string
+import random
+import math
```

Comment:
I haven't done any benchmarking yet and this isn't a big deal but is there a specific reason `math.floor`/`math.log` is used as opposed to `numpy.floor`/`numpy.log`?

Reply:
the `numpy.log` is based on e, `math.log` can change the base. `math.floor` is quicker when accepting a single value as parameter compared to `numpy.floor`